SourceKitStressTester/Sources/SwiftSourceKit/SourcekitdResponse.swift

```diff
@@ -234,6 +238,20 @@ public class SourceKitdResponse: CustomStringConvertible {
     return value.getOptional(.key_Notification) != nil
   }
 
+  public var isSemaDisabledNotification: Bool {
+    if isNull { return false }
+    guard let notification = value.getOptional(.key_Notification)?.getUID()
+    else { return false }
```

Comment:
Could factor this out into `notificationType` but there's only two, so not too bad right now.